CMakeLists.txt

```diff
@@ -98,19 +108,84 @@ if(BUILD_FRAMEWORK)
         MACOSX_FRAMEWORK_SHORT_VERSION_STRING ${PROJECT_VERSION}
         MACOSX_FRAMEWORK_BUNDLE_VERSION ${PROJECT_VERSION}
         XCODE_ATTRIBUTE_INSTALL_PATH "@rpath"
-        OUTPUT_NAME Ogg
+		OUTPUT_NAME Ogg
```

Comment:
You are using tabs here instead of spaces for indentation, even though everywhere else it seems spaces are used.
I would prefer to have it consistent and use spaces everywhere.